lib/active_model_serializers.rb

```diff
@@ -1,14 +1,16 @@
-require 'logger'
 require 'active_model'
 require 'active_support'
+require 'active_support/tagged_logging'
+require 'active_support/logger'
```

Comment:
I think we can remove these 2 requires, since we already require `active_support`